tutors3/plugin.py

```diff
@@ -15,7 +15,7 @@
         "USE_SSL": True,
         "STORAGE_BUCKET": "openedx",
         "FILE_UPLOAD_BUCKET": "{{ S3_STORAGE_BUCKET }}",
-        "PROFILE_IMAGE_BUCKET": "{{ S3_STORAGE_BUCKET }}",
+        "PROFILE_IMAGE_BUCKET": "",
```

Comment:
Hold on, one more tiny nit. Since this default changes here, we'll need the corresponding bit in the `README.md` changed as well.

Reply:
Yeah! I will add a `docs` commit with it.

It's almost over! LoL

Reply:
That's actually not necessary. A `docs:` commit is for when you're *only* updating the documentation, or adding new documentation material, as you did in #39.

When you make functional changes, all documentation changes related to that functional change should be in the same commit. :)